Schema/PublicSchema/RemoteDependencyData.bond

```diff
@@ -0,0 +1,53 @@
+import "Domain.bond"
+
+namespace AI
+
+[Description("An instance of Remote Dependency represents an interaction of the monitored component with a remote component/service like SQL or an HTTP endpoint.")]
+struct RemoteDependencyData
+    : Domain
+{
+    [Description("Schema version")]
+    10: required int32 	 ver = 2;
+    
+    [MaxStringLength("1024")]
+    [Description("Name of the command initiated with this dependency call. Low cardinality value. Examples are stored procedure name and URL path template.")]
+    20: required string 	 name;
+    
+    [MaxStringLength("128")]
+    [Description("Identifier of a dependency call instance. Used for correlation with the request telemetry item corresponding to this dependency call.")]
+    30: string 	 id;
+    
+    [MaxStringLength("1024")]
+    [Description("Result code of a dependency call. Examples are SQL error code and HTTP status code.")]
+    40: string 	 resultCode;
+    
+    [CSType("TimeSpan")]
+    [Description("Request duration in TimeSpan 'G' (general long) format: d:hh:mm:ss.fffffff.")]
+    [ActAsRequired("Renaming value to duration.")]
+    61: required string 	 duration;
+    
+    [Description("Indication of successfull or unsuccessfull call.")]
+    120: nullable<bool> 	 success = true;
```

Comment:
Why would success be nullable?  Do we not always know if the call succeeded or not?